pages/common/gpg-tui.md

```diff
@@ -0,0 +1,36 @@
+# gpg-tui
+
+> Terminal user interface for GNU Public Guard.
+> More information: <https://github.com/orhun/gpg-tui>.
+
+- Start `gpg-tui`:
+
+`gpg-tui`
+
+- Start `gpg-tui` enabling color and ASCII armored output:
+
+`gpg-tui --style {{colored}} --armor`
+
+- Show help in `gpg-tui`:
+
+`?`
+
+- Quit `gpg-tui`:
```

Comment:
```suggestion
- Quit:
```

Reply:
`Quit` on it's own seems extremely short to be a example description

Reply:
I disagree with this, too short, it just adds a tiny bit of data which could be helpful